lib/openhab/dsl/types/point_type.rb

```diff
@@ -148,7 +162,21 @@ def coerce_single(other)
             other.state
           elsif other.respond_to?(:to_str)
             PointType.new(other.to_str)
+          elsif other.respond_to?(:to_hash)
+            PointType.new(other.to_hash)
+          end
+        end
+
+        #
+        # Convert possible keyword arguments into ordered arguments for constructor
+        #
+        def from_kargs(kargs)
+          keys = [%i[lat long alt], %i[latitude longitude altitude]]
+          keys.each do |key_set|
+            values = kargs.transform_keys(&:to_sym).values_at(*key_set).compact
+            return *values if values.length >= 2
```

Comment:
Oh yeah, that's what you're trying to do here. But it's not going to work right, cause if you pass say latitude and _altitude_, you'll have two values, but altitude will be interpreted as longitude.